media-source-respec.html

```diff
@@ -3161,7 +3428,65 @@ <h2>Examples</h2>
         </div>
       </div>
     </section>
+    <h3>Use of the Managed Media Source Extensions</h3>
+    <pre class="example html">
+      &lt;script&gt;
+      async function setUpVideoStream() {
+        // Specific video format and codec
+        const mediaType = 'video/mp4; codecs="mp4a.40.2,avc1.4d4015"';
+
+        // Check if the type of video format / codec is supported.
+        if (ManagedMediaSource?.isTypeSupported(mediaType)) {
```

Comment:
Unless I'm seriously mistaken, it looks like the result of this "if" statement is the opposite of what is wanted:
It currently is like, "if ManagedMediaSource is defined and has a method isTypeSupported that, when called with mediaType, returns true, then fail."
aside: nice optional chaining usage, though :)

Reply:
> aside: nice optional chaining usage, though :)

Unfortunately, optional chaining doesn't work here... If a global variable named `ManagedMediaSource` doesn't exist, then this will still throw a `ReferenceError`:
![image](https://github.com/w3c/media-source/assets/649348/b42d079e-ed80-4cf3-ba8d-ea74f5d708b9)

Reply:
@MattiasBuelens You're correct. Maybe instead:
if (!this.ManagedMediaSource??.isTypeSupported(mediaType)) { /* Either MMSE itself isn't supported or MMSE is supported but doesn't support mediaType */ }

Reply:
I went with `window`